nifi-provenance-reporting-tasks/pom.xml

```diff
@@ -42,14 +44,14 @@
             <version>1.21.0</version>
         </dependency>
         <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-            <version>2.12.7.1</version>
+            <groupId>co.elastic.clients</groupId>
+            <artifactId>elasticsearch-java</artifactId>
+            <version>7.17.10</version>
```

Comment:
maybe there exists a newer version of the `elasticsearch-java` client that could be used?